docs/quickstart/assessment/standalone_assessment.py

```diff
@@ -47,56 +57,443 @@
+                                    "type": "Grade",
+                                    "name": "5"
+                                },
+                                {
+                                    "type": "Subject",
+                                    "name": ["Math", "Science"]
+                                }
+                            ],
+                            "none": [
+                                {
+                                    "type": "Grade",
+                                    "name": "6"
+                                }
+                            ]
+                        }
+                    }
+                }
+            },
+            "dependencies": {
+                "question_editor_api": {
+                    "init_options": {}
+                },
+                "questions_api": {
+                    "init_options": {}
+                }
+            },
+            "widget_templates": {
+                "back": True,
+                "save": True,
+                "widget_types": {
+                    "default": "questions",
+                    "show": True
+                }
+            },
+            "container": {
+                "height": "auto",
+                "fixed_footer_height": 0,
+                "scroll_into_view_selector": "body"
+            },
+            "label_bundle": { 
+                
+                "backButton": "Zurück",
+                "loadingText": "Wird geladen",
+                "modalClose": "Schließen",
+                "saveButton": "Speichern",
+                "duplicateButton": "Duplikat",
+
+                
+                "dateTimeLocale": "en-us",
+                "toolTipDateTimeSeparator": "um",
+                "toolTipDateFormat": "DD-MM-YYYY",
+                "toolTipTimeFormat": "HH:MM:SS",
+            }
+        },
+    }
+
+# Reports API configuration parameters.
+report_request = {
+    "reports" : [{
+        "id": "session-detail",
+        "type": "session-detail-by-item",
+        "user_id": "906d564c-39d4-44ba-8ddc-2d44066e2ba9",
+        "session_id": "906d564c-39d4-44ba-8ddc-2d44066e2ba9"
+    }]
+}
+
+# Question Editor API configuration parameters.
+question_editor_request = {
+    "configuration" : {
+       "consumer_key": config.consumer_key,
+    },
+    "widget_conversion": True,
+    "ui" : {
+        "search_field" : True,
+    },
+    "layout":{
+        "global_template": "edit_preview",
+        "mode": "advanced"
+    }
+    }
 
 # Set up Learnosity initialization data.
-init = Init(
-    'items', security, config.consumer_secret,
-    request=items_request
+initItems = Init(
+    "items", security, config.consumer_secret,
+    request = items_request
+)
+
+initQuestions = Init(
+    "questions", security, config.consumer_secret,
+    request = questions_request
 )
-generated_request = init.generate()
+
+initAuthor = Init(
+    "author",security,config.consumer_secret,
```

Comment:
what I mean here was "space after comma".
this apply for all other Init below

Reply:
fixed